apps/fas-jupyter-engsci201/form.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Jupyter Notebook - Eng Sci 201
+cluster: "academic"
+attributes:
+  modules: null
+  extra_jupyter_args: ""
+  course: atg_general
+  custom_time:
+    label: "Allocated Time (expressed in MM , or HH:MM:SS , or DD-HH:MM)."
+    value: "04:00:00"
+    widget: text_field
+  bc_queue: "academic"
+  custom_email_address:
+    label: email address for status notification
+    widget: text_field
+  
+  custom_memory_per_node: 16
+  
+  
+  custom_num_cores: 8
+  
+  custom_num_gpus: 0 
+  jupyter_version: harvardat_jupyter-engsci201_sha-3e5a86a.sif
```

Comment:
Since we moved repositories, this should be `huitacademictechnology_jupyter-engsci201_sha-3e5a86a.sif`. You can see the image here: https://hub.docker.com/repository/docker/huitacademictechnology/fas-jupyter-engsci201/general

Reply:
Sounds good. Will go ahead and update that jupyter_version!